pkg/commands/ls.go

```diff
@@ -113,7 +113,14 @@ func mergeStagingComponents(deployed []apis.ComponentMeta, env *types.EnvMeta, i
 					v.Traits[j] = t
 				}
 				aspec.Components[i] = v
+				/* Before any traits is attached to the workload, if ApplicationConfiguration is retrieved through
+				OpenAPI, the .spec.Components[].Traits is nil; if it's resembled by local file, .spec.Components[].Traits
+				is the []v1alpha2.ComponentTrait, 0). Let's set it to be equal before comparing the whole .Spec field.*/
```

Comment:
> is the []v1alpha2.ComponentTrait, 0)

What does this mean?

Reply:
It should be `make([]v1alpha2.ComponentTrait, 0)`:)